atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2441,8 +2440,7 @@ void WebContents::CapturePage(mate::Arguments* args) {
 
   host->GetView()->CopyFromSurface(gfx::Rect(rect.origin(), view_size),
       bitmap_size,
-      base::Bind(&OnCapturePageDone, callback),
-      kBGRA_8888_SkColorType);
+      base::Bind(&OnCapturePageDone, callback));
```

Comment:
Bind-> BindOnce for `base::OnceCallback` arg changes?

Reply:
Sure. Fixed at https://github.com/brave/muon/commit/d1df1be286ad93e34a79b12738b3f6e838f76474